### ISSUE TITLE
We're showing the upload icon when the feature is disabled

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#36799]
### Describe the bug

A customer just told us that the icon is there when the feature is disabled. We shouldn't show the icon in that case
![image](https://github.com/metabase/metabase/assets/1711649/65653b02-ce60-46eb-b95e-39b077142526)

### To Reproduce

Just go into a collection with upload csv's disabled

### Expected behavior

we shouldn't show the icon

### Information about your Metabase installation

```JSON
v47.x
```

### Severity

P3

**Comment by nemanjaglumac** [comment]
This seems to have been a conscious decision @paoliniluis 

Please take a look at https://github.com/metabase/metabase/pull/31345
cc @iethree

**Comment by paoliniluis** [comment]
Absolutely, this issue is just there to flag that a customer reported it and, if many users do, we should remove the icon

**Comment by paoliniluis** [comment]
change it to a new feature request

**Comment by brunobergher** [comment]
From what I can gather there was only this one instance, so I propose we close this.
